### PR TITLE
Remove dep on weft

### PR DESCRIPTION
**Pull request by ruhan1**
Remove the dep on weft and o11yphant core so that it can be safely used by micro service.

### CODE REVIEW

File: storage/src/main/java/org/commonjava/storage/pathmapped/core/PathMappedFileManager.java [hunk 1]
```diff
@@ -71,8 +70,7 @@ public PathMappedFileManager( PathMappedStorageConfig config, PathDB pathDB, Phy
             logger.info( "Start path-mapped GC thread, gcIntervalInMinutes: {}", gcIntervalInMinutes );
             int priority = 4;
             String name = "path-mapped-gc";
-            gcThreadPool = Executors.newScheduledThreadPool( 1, new NamedThreadFactory( name, new ThreadGroup( name ),
-                                                                                        true, priority ) );
+            gcThreadPool = Executors.newScheduledThreadPool( 1 );
```
Comment (sswguo) [inline storage/src/main/java/org/commonjava/storage/pathmapped/core/PathMappedFileManager.java:73]: Do we still need to identify the thread by the name ? if then, we can copy the NamedThreadFactory from weft.

Comment (ruhan1) [reply storage/src/main/java/org/commonjava/storage/pathmapped/core/PathMappedFileManager.java:73]: I guess not need. I think we will disable the gc thread from indy and only keep it in the store service. We only need one gc working anyway.

Comment (sswguo) [reply storage/src/main/java/org/commonjava/storage/pathmapped/core/PathMappedFileManager.java:73]: See, LGTM.

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**